package.json

```diff
@@ -182,6 +182,11 @@
 					],
 					"default": "colour",
 					"description": "Specifies the colour theme of the icon displayed on the Git Graph tab."
+				},
+				"git-graph.retainContextWhenHidden": {
+					"type": "boolean",
+					"default": false,
+					"description": "Controls if the Git Graph View is kept around even when the panel is no longer visible (e.g. moved to background tab)."

```

Comment:
Could this please be reworded to: `Specifies if the Git Graph view Visual Studio Code context is kept when the panel is no longer visible (e.g. moved to background tab). Enabling this setting will make Git Graph load significantly faster when switching back to the Git Graph tab, however has a higher memory overhead.`.

The first sentence is more consistent with the other property descriptions, and adding the second sentence highlights the pro's and con's of using this setting.